graylog2-server/src/main/java/org/graylog/integrations/aws/inputs/AWSInput.java

```diff
@@ -190,7 +194,29 @@ public ConfigurationRequest combinedRequestedConfiguration() {
                     ConfigurationField.Optional.OPTIONAL,
                     NumberField.Attribute.ONLY_POSITIVE));
 
+            request.addField(getOverrideSourceFieldDefinition());
+
             return request;
         }
     }
+
+    public static TextField getOverrideSourceFieldDefinition() {
+        return new TextField(
+                CK_OVERRIDE_SOURCE,
+                "Override Source",
+                "",
+                "The source is set to the Kinesis message by default. Set this if you want to override it with a custom value.",
```

Comment:
Suggestion: `"The source is set to '%s' by default. Set this if you want to override it with a custom value.".formatted(KinesisRawLogCodec.SOURCE)`